javaslang/src-gen/main/java/javaslang/API.java

```diff
@@ -84,6 +84,280 @@ private API() {
     }
 
     //
+    // Aliases for static factories
+    //
+
+    /**
+     * Alias for {@link Function0#of(Function0)}
+     *
+     * @param <R>             return type
+     * @param methodReference A method reference
+     * @return A {@code Function0}
+     */
+    public static <R> Function0<R> function0(Function0<R> methodReference) {
+        return Function0.of(methodReference);
+    }
+
+    /**
+     * Alias for {@link Function1#of(Function1)}
+     *
+     * @param <R>             return type
+     * @param <T1>            type of the 1st argument
```

Comment:
`R` is not return type of this function, but it return type in the lambda. I am use same text as in JavaDoc of `javaslang.Function1#of`.
All types in this JavaDoc are refers to the argument types
I will be glad if you offer a good option